packages/node_modules/@webex/plugin-meetings/src/media/properties.js

```diff
@@ -61,6 +61,14 @@ export default class MediaProperties {
   }
 
   setRemoteStream(remoteStream) {
+    const stream = this.remoteStream;
+    const isCompatible = stream && stream.getTracks && stream.addTrack;
```

Comment:
This is check is mainly for[ IE and Opera for Android](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream/getTracks#Browser_Compatibility) which doesn't support `MediaStream.getTracks()`. In case someone ran Opera on Android the code would still function and not throw errors.

Reply:
Right, not questioning the check but rather what is being checked against. Should you be checking on `remoteStream` and not `this.remoteStream`?

Reply:
I'm invoking `.getTracks()` from `this.remoteStream`, and I'm also checking to see if a previous stream already exists. I'm getting tracks from one stream and combining it with the other stream. Previously it just replaced the streams which caused a strange bug in that the remote audio never ended.  If you want I can alter the code to:

```suggestion
    const isCompatible = stream && stream.getTracks && remoteStream.addTrack;
```